frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -550,7 +550,7 @@ Book view settings, reading progress, highlights, bookmarks and notes (collectiv
 You can decide between two locations where these will be saved:
 - alongside the book file itself (the long time default): these sdr folders will be visible when you browse your library directories with another file browser or from your computer, which may clutter your vision of your library. But this allows you to move them along when you reorganize your library, and also survives any renaming of parent directories. Also, if you perform directory synchronization or backups, your settings will be part of them.
 - all inside koreader/docsettings/: these sdr folders will only be visible and used by KOReader, and won't clutter your vision of your library directories with another file browser or from your computer. But any reorganisation of your library (directories or filename moves and renamings) may result in KOReader not finding your previous settings for these books. These settings won't be part of any synchronization or backups of your library.
-- all inside koreader/hashdocsettings/ as hashes: sdr folders are identified not by filepath and filename but by partial MD5 hash. This allows you to rename, move, and copy documents at will and and KOReader will link them to their metadata. However, any file modifications such as writing highlights into PDFs or downloading from Calibre may change the hash, resulting in metadata being unlinked from the file. Calculating file hashes may also impact loading times.]])
+- all inside koreader/hashdocsettings/ as hashes: sdr folders are identified not by filepath and filename but by partial MD5 hash. This allows you to rename, move, and copy documents outside at will outside of KOReader, and KOReader will still link them to their metadata. However, any file modifications such as writing highlights into PDFs or downloading from Calibre may change the hash, resulting in metadata being unlinked from the file. Calculating file hashes may also impact loading and file browser browsing times. This option may be preferable for users with many documents that want to remove sdr folder clutter or who might have multiple copies of their documents in different directory paths or devices.]])
```

Comment:
- _copy documents outside at will outside of KOReader_ you removed the duplicated `and`, but introduced duplicated `outside` :)

> _This option may be preferable for users with many documents that want to remove sdr folder clutter or who might have multiple copies of their documents in different directory paths or devices._

Feels a bit too generic: we are many with many documents :) - and those who don't like sdr folder clutter could use the 2nd option. I don't think "having multiple copies in different directory paths" is a use case.
What feels like THE use case is people synchronising settings across multiple devices _of different brand_ (or different directories layout/Library root location).
Dunno how to word that :)

Isn't this big text too long? Does it display and read well with the small font size it must get?

Reply:
I agree it's too long now, that's why I hadn't added use cases originally haha. In latest push I shortened the use cases as you mentioned and tried to shorten a bit more, but I'm not sure how else to shorten this large 3-bullet string.